### ISSUE TITLE
Option of clicking in the outline to take the item to the top  of the page

### ISSUE THREAD

**Issue by squinonescolon** [superlou/fountain-atom#87]
Would it be possible to have an option to take the selected item in the outline to the top of the page instead of the bottom?

**Comment by jrbostic** [comment]
You can already drag outline elements up and down, but they are locked down by default... Has to be toggled via a button above.  Is that what you're talking about?

**Comment by squinonescolon** [comment]
No. When you click on an item in the outline the text in the file scrolls to that position on file but it gets aligned to the bottom of the page, hiding what you want to read. It would make more sense to align the text to the top of the page making the material you want to read instantly visible. 

Steven Quiñones Colón
VFX Producer
939.240.1006

**Comment by jrbostic** [comment]
I see.  That sounds like a generally desirable default behavior.  I can't check at the moment, so correct me if I'm wrong, but I'm thinking the final scroll position might be effected by your initial relative position in the document.  So, if you start out above the section, then it would demonstrate your described behavior, but if you start out below it would actually appear to do what you want.  

So, there's an event handler that manages the scroll behavior and it uses some atom built in functions to bring the index into view.  First step would be to see if there's an existing flag or something that provides this kind of behavior.  If there's nothing like that, the second item would be to roll some custom code that calculates the offset based on current relative position and window height.

**Comment by squinonescolon** [comment]
Indeed that's what's happening. The default desired behavior should be, in
my opinion, that it would scroll to the top or the middle. Scroll to the
bottom is the worst possible option. I could not find anything in the
preferences for Atom that would change that.

Steven Quiñones Colón
VFX Producer
steven@stevenquinones.com
http://www.imdb.com/name/nm1167131/
https://www.linkedin.com/in/stevenquinonescolon/
https://vimeo.com/257703564
939.240.1006
